src/rules/html/warn-on-remote-script.js

```diff
@@ -0,0 +1,25 @@
+import { VALIDATION_WARNING } from 'const';
+import { isStrictRelativeUrl } from 'schema/formats';
+import * as messages from 'messages';
+
+
+export function warnOnRemoteScript($, filename) {
+  return new Promise((resolve) => {
+    var linterMessages = [];
+
+    $('script').each((i, element) => {
+      const src = $(element).attr('src');
+
+      if (src !== undefined && !isStrictRelativeUrl(src)) {
```

Comment:
This should also respect `src` that starts with `moz-extension://`. Please also add this to the test.